compositor_render/src/wgpu/format/rgba_to_yuv.wgsl

```diff
@@ -23,26 +23,35 @@ fn vs_main(input: VertexInput) -> VertexOutput {
 
 var<push_constant> plane_selector: u32;
 
+fn linear_to_srgb(linear: vec3<f32>) -> vec3<f32> {
```

Comment:
I have to do this in shader because WebGl does not support creating non srgb views on srgb textures.
We could do this better if we switch to WebGPU only